frontend/apps/reader/modules/readerpagemap.lua

```diff
@@ -82,6 +82,7 @@ function ReaderPageMap:_postInit()
         self:resetLayout()
         self.view:registerViewModule("pagemap", self)
         if self.ui.document.is_new and self.has_pagemap_document_provided
+                and G_reader_settings:isTrue("pagemap_notify_document_provided")
                 and not (self.use_page_labels or self.show_page_labels) then
```

Comment:
Now that it's optionnal and default to off, may be remove that `not (use or show)` condition ? (So one can see the source when he is using them).

Reply:
Or, in this case show InfoMessage instead of ConfirmBox.